WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Metadata/OrderCustomFieldsDetails.swift

```diff
@@ -0,0 +1,52 @@
+import Yosemite
+import SwiftUI
+
+struct OrderCustomFieldsDetails: View {
+    let customFields: [OrderMetaData]
+
+    var body: some View {
+        NavigationView {
+            GeometryReader { geometry in
+
+                Color(.listBackground).edgesIgnoringSafeArea(.all)
+
+                VStack(alignment: .leading, spacing: 10) {
+                    ForEach(customFields, id: \.self) { customField in
+                        CustomFieldView(customField: customField)
+                    }
+                }
+            }.navigationTitle("Custom Fields")
+        }
+
+    }
+}
+
+private struct CustomFieldView: View {
+    let customField: OrderMetaData
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 10) {
+                    Divider()
+
+            Text(customField.key)
+                        .headlineStyle()
+                        .padding([.leading, .trailing])
+
+                    HStack(alignment: .bottom) {
+                        Text(customField.value)
+                            .bodyStyle()
+                            .frame(maxWidth: .infinity, alignment: .leading)
+                    }
+                    .padding([.leading, .trailing])
+
+                    Divider()
+                }
+                .background(Color(.basicBackground))
+    }
```

Comment:
I just took a quick peek at this, and I'm wondering if you considered using the `TitleAndValueRow` component for this? Since wp-admin shows the key and value side-by-side, I was imagining something more like that here.

Either way, would it be helpful to have a quick design mockup to make sure we're all on the same page?

Reply:
Good point! I replaced this part and adopted the `TitleAndSubtitleRow` as you suggested following your discussion on slack!